src/Nodes/Shapes/SVGPath.php

```diff
@@ -80,7 +80,7 @@ public function rasterize(SVGRasterizer $rasterizer): void
     private static function getPathParser(): PathParser
     {
         if (!isset(self::$pathParser)) {
```

Comment:
I think we can get rid of the `if (!isset(self::$pathParser))` entirely now, right? :)